app/components/candidate_interface/rejection_reasons_component.rb

```diff
@@ -24,7 +29,7 @@ def course_details_row(application_choice)
     end
 
     def course_details_row_value(application_choice)
-      if EndOfCycleTimetable.find_down?
+      if EndOfCycleTimetable.find_down? || FeatureFlag.active?(:structured_reasons_for_rejection)
```

Comment:
Sorry if I'm being dense, but why don't we want to link to Find? I thought we were only concerned with that between cycles when Find is down.

Reply:
➕

Reply:
@paulrobertlloyd did I interpret the requirement correctly here about removing the link to Find? What is the thinking behind this change?

Reply:
I have no idea what this is about! Where did this requirement surface?

Reply:
@paulrobertlloyd It was the third sub-task on the card
> **See feedback from your previous application** Only shown the **Course** (not linked to Find), **Status**, and **Feedback** in each summary card.

Reply:
Huh. So funny reading my words back several months later! Also funny that I wrote this and yet it’s not reflected in the prototype. Funnier still that it took me a while to figure out what was wrong with seeing the link. Or maybe not that funny at all…

Given the above, let’s reduce complexity, remain consistent, and maintain the link. We can unlink it if it proves a problem (I suspect I may have been overthinking the solution).

Reply:
That link has been reinstated now.